### PR TITLE
docs: add reference to the dummy plugin implementations

### PR DESCRIPTION
**Pull request by vigsterkr**
# Description

This just extends the plugin documentation section with the dummy/example plugin repositories:
 - https://github.com/SwissDataScienceCenter/renku-dummy-provider
 - https://github.com/SwissDataScienceCenter/renku-dummy-annotator
 - https://github.com/SwissDataScienceCenter/renku-cli-plugin-example
 - https://github.com/SwissDataScienceCenter/renku-dummy-format

note before merging this PR the linked repositories visibility should be changed to public.

### CODE REVIEW

File: docs/reference/plugins.rst [hunk 1]
```diff
@@ -82,6 +84,25 @@ where `myproject.pluginmodule:mycmd` points to a click command e.g.:
...
+
+.. code-block:: console
+
+   $ renku workflow export --format cwl <my_workflow>
+
+.. autoclass:: renku.core.models.workflow.converters.IWorkflowConverter
+    :members:
+
+We created a `dummy <https://github.com/SwissDataScienceCenter/renku-dummy-format>`_ implementation of such
+converter plugin.
```
Comment (Panaetius) [inline docs/reference/plugins.rst:104]:
```suggestion
a converter plugin.
```

**Review by Panaetius — CHANGES_REQUESTED** [review]
Thank you! Just a minor spelling change, otherwise this is a great first step!

**Review by Panaetius — APPROVED** [review]

**Comment by vigsterkr** [comment]
@rokroskar just in case you are pinged as well

**Comment by rokroskar** [comment]
Just seeing this now, sorry! Looks great - I would maybe call them "examples" rather than "dummies" but that's my personal preference 😆

**Pull request merged**